git_calendar/build.py

```diff
@@ -20,6 +20,7 @@ def main(argv=sys.argv[1:]):
     parser.add_argument('inputs', nargs='+', help="input files", type=Path)
     parser.add_argument('--output', '-o', help="output directory", type=str)
     parser.add_argument('--index', '-i', help="output index file", type=str)
+    parser.add_argument('--content', '-c', help="output HTML content to be included in other pages", type=str)
```

Comment:
`--html-content` or `--html-body` maybe, since "content" itself, before understanding what this did, made me think it was about the content of the data.  But I have no objections to this as it is.

Reply:
Good suggestion. I will adapt.